doc/weakly_supervised.rst

```diff
@@ -327,18 +348,17 @@ Side-Information, Xing et al., NIPS 2002
 
 `MMC` minimizes the sum of squared distances between similar examples, while
 enforcing the sum of distances between dissimilar examples to be greater than a
-certain margin. This leads to a convex and, thus, local-minima-free
+certain margin, default is 1. This leads to a convex and, thus, local-minima-free
```

Comment:
we do not provide a way to change this margin (which is only a scaling factor anyway), so please simply say greater than 1.